src/resources/views/crud/columns/range.blade.php

```diff
@@ -0,0 +1,24 @@
+{{-- regular object attribute --}}
```

Comment:
Ugh.... you won't like this Munjal but... I think this has gone in a wrong direction 👀 I mean... it's pretty... but I don't think the `range` column should be shown as a progress bar... I think it'd just be better to echo it out as text (load the text column).

Why do I say think? Well 
- for the `range` field it makes sense to show a range input. It forces the user to pick between two limits... and it's easy to just drag to get a valuel;
- for the column, I don't think it's useful to see a progress bar; I think what you want most times is just to see the value; otherwise you'd be looking at an attribute like `price` or `priority` and see a progress bar... without seeing the actual value; not what you want, I think;

So let's just load the text column here, please. 

Alternatively, if you think I'm wrong on this... we could at least use the progress bar design in https://backstrap.net/base/progress.html and by that I mean this one:

<img width="1256" alt="CleanShot 2022-10-11 at 11 30 50@2x" src="https://user-images.githubusercontent.com/1032474/195039691-7d5544d4-2cfc-4be6-b410-0a17c2a8ba6b.png">

Which allows us to have both the idea of progress bar, and the actual value.

What do you think?

Reply:
Applied `progress` changes as requested

Reply:
Ok. Noted & working on the same..

Reply:
1) Number added on range.
2) Dynamic class added.
3) Stripe option added (is_striped [1/0]).
Please review & let me know.

Reply:
Excellent. Good progress here @munjaldevelopment !

One important thing I noticed - the value of this column is interpreted as a range between 1 and 100. Which might not be the case at all. It's used directly as a percentage for the width. And its value is output directly as x%. That's wrong. Because 
- yes, a person can use this field/column as a range between 1 and 100; in which case this will match perfectly with the CSS and the % will make sense; 25 out of 100 is 25%;
- but no, a person can ALSO use this field/column as a range between 1 and 10; and if the value is 5, it does NOT mean 5%;

What I'm saying here is that 
- [ ] we should take that into account, and calculate the proper percentage out of the max value, then use the percentage for the CSS;
- [ ] do NOT output `x%` on top of the progress bar, but instead output simply `x`